app/main.py

```
@@ -31,13 +33,18 @@
 import httpx
 from app.core.config import SUPABASE_URL, SUPABASE_KEY, HEALTH_CHECK_TIMEOUT, SUPABASE_TABLE, emit_startup_warnings
 from app.models.dto import HealthResponse
+from app.core.scheduler import start_scheduler, shutdown_scheduler
+from app.services.monitoring_service import send_discord_report
+
 
 @asynccontextmanager
 async def lifespan(app: FastAPI):
```

Ruff (0.15.2)

[warning] Unused function argument: `app`

(ARG001)